packages/new-polymath-issuer/src/pages/SecurityTokensDividends/Presenter.tsx

```diff
@@ -190,10 +190,7 @@ export const Presenter: FC<Props> = ({
             <Grid>
               <FormItem name="walletAddress">
                 <FormItem.Label>Wallet Address</FormItem.Label>
-                <FormItem.Input
-                  component={TextInput}
-                  placeholder="Wallet address"
-                />
+                <FormItem.Input component={TextInput} />
```

Comment:
Why remove the placeholder?

Reply:
Did this without noticing, adding it back